<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtsMonitorZc</class>
 <widget class="QWidget" name="AtsMonitorZc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请选择消息包：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>ATS时间信息</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSR验证命令帧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSR执行命令帧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上电TSR确认帧</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>到</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="startmonitor">
        <property name="text">
         <string>开始监测</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>307</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="firstbutton">
     <property name="text">
      <string>首页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="prebutton">
     <property name="text">
      <string>上一页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="page">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="nextbutton">
     <property name="text">
      <string>下一页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="lastbutton">
     <property name="text">
      <string>尾页</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="totalcount">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
